source/guides/maintenance/cpu-performance.rst

```diff
@@ -193,6 +193,53 @@ group:
    After adding a new group, you must log out and log back in for the new group
    to take effect.
 
+Thermal configuration
+=====================
+
+Better thermal control and performance can be archived by providing platform
+specified configuration to thermald.
+
+`Linux DPTF Extract Utility`_ is a companion tool to thermald, This tool can
+make use of "Intel ® Dynamic Platform and Thermal Framework (`Intel DPTF`_)"
+technology, and convert to the thermal_conf.xml configuration format used
+by thermald.
+
+First, make sure thermald version is equal or above 1.9.1:
+
+.. code:: bash
+
+   thermald --version
+
+Otherwise, you need to add an option :command:`--ignore-default-control` to
+thermald:
+
+.. code:: bash
+
+   # edit thermald.servcie
+   vim /usr/lib/systemd/system/thermald.service
```

Comment:
```suggestion
sudo  vim /usr/lib/systemd/system/thermald.service
```

Reply:
See comment above about avoiding this whole section, but in Clear Linux editing file under `/usr/` is not safe and prone to being overwritten by system update. See the documentation on stateless. 

If you have to do this in the future on Clear Linux system, a better approach is to create a modified service file in `/etc/`. The `systemctl edit --full thermald.service` can help do this neatly. Even better would be a drop-in directory for the unit. https://www.freedesktop.org/software/systemd/man/systemd.unit.html